### ISSUE TITLE
generate-playbook: gracefully handle koji lib < 1.18

### ISSUE THREAD

**Issue by ktdreyer** [ktdreyer/koji-ansible#92]
If we are using a Koji library older than 1.18, `session.multicall` is not callable. We get an error like so:

```
$ ./utils/generate-playbook  --profile foobar --regex myproduct-17.0
Traceback (most recent call last):
  File "./utils/generate-playbook", line 226, in <module>
    main()
  File "./utils/generate-playbook", line 221, in main
    generate_tags_playbook(session, args.regex)
  File "./utils/generate-playbook", line 199, in generate_tags_playbook
    tags = search(session, 'tag', 'getTag', regex)
  File "./utils/generate-playbook", line 187, in search
    with session.multicall(strict=True, batch=100) as m:
TypeError: 'bool' object is not callable
```

We should detect this early and bail `if not callable(session.multicall))`.

**Comment by ktdreyer** [comment]
I've implemented this in https://github.com/ktdreyer/koji-ansible/pull/96

**Comment by ktdreyer** [comment]
I've merged #96 to master now.
